changelog.md

```diff
@@ -334,3 +334,7 @@ Breaking changes:
 * Fix: Semver version ranges for patches
 * Refactor `lib/errors`
 * Increase tests timeout to 30 seconds
+
+## 1.0.1
```

Comment:
Could you possible change this line to `## Next`? I add the version tag to changelog when I release. Anyway, I think this qualifies for a `1.1.0` release!